maven-java/kalix-maven-archetype-event-sourced-entity/README.md

```diff
@@ -1,6 +1,6 @@
 # kalix-maven-archetype-event-sourced-entity
 
-This archetype can be used to generate a project suitable for a service using an Event-sourced Entity with [Kalix](https://www.lightbend.com/akka-serverless).
```

Comment:
this is pointing to the product page at lightbend. Moving to the the Kalix docs directly.